src/pynwb/testing/mock/icephys.py

```diff
@@ -0,0 +1,207 @@
+        resolution=resolution,
+        starting_time=starting_time,
+        rate=rate,
+        electrode=electrode or mock_IntracellularElectrode(),
+        gain=gain,
+        capacitance_slow=capacitance_slow,
+        resistance_comp_correction=resistance_comp_correction,
+    )
+
+
+def mock_CurrentClampSeries(
+    name=None,
+    data=None,
+    electrode=None,
+    gain=0.02,
+    stimulus_description="N/A",
+    bias_current=None,
+    bridge_balance=None,
+    capacitance_compensation=None,
+    resolution=-1.0,
+    conversion=1.0,
+    timestamps=None,
+    starting_time=None,
+    rate=100_000.,
+    comments="no comments",
+    description="no description",
+    control=None,
+    control_description=None,
+    sweep_number=None,
+    offset=0.0,
+    unit="volts",
+):
+    return CurrentClampSeries(
+        name=name if name is not None else name_generator("CurrentClampSeries"),
+        data=data if data is not None else np.ones((30,)),
+        electrode=electrode or mock_IntracellularElectrode(),
+        gain=gain,
+        stimulus_description=stimulus_description,
+        bias_current=bias_current,
+        bridge_balance=bridge_balance,
+        capacitance_compensation=capacitance_compensation,
+        resolution=resolution,
+        conversion=conversion,
+        timestamps=timestamps,
+        starting_time=starting_time,
+        rate=rate,
+        comments=comments,
+        description=description,
+        control=control,
+        control_description=control_description,
+        sweep_number=sweep_number,
+        offset=offset,
+        unit=unit,
+    )
+
+
+def mock_CurrentClampStimulusSeries(
+    name=None,
+    data=None,
+    electrode=None,
+    gain=0.02,
+    stimulus_description="N/A",
+    resolution=-1.0,
+    conversion=1.0,
+    timestamps=None,
+    starting_time=None,
+    rate=100_000.,
+    comments="no comments",
+    description="no description",
+    control=None,
+    control_description=None,
+    sweep_number=None,
+    offset=0.0,
+    unit="amperes",
+):
+    return CurrentClampStimulusSeries(
+        name=name or name_generator("CurrentClampStimulusSeries"),
+        data=data if data is not None else np.ones((30,)),
+        electrode=electrode or mock_IntracellularElectrode(),
+        gain=gain,
+        stimulus_description=stimulus_description,
+        resolution=resolution,
+        conversion=conversion,
+        timestamps=timestamps,
+        starting_time=starting_time,
+        rate=rate,
+        comments=comments,
+        description=description,
+        control=control,
+        control_description=control_description,
+        sweep_number=sweep_number,
+        offset=offset,
+        unit=unit,
+    )
+
+
+def mock_IZeroClampSeries(
+    name=None,
+    data=None,
+    electrode=None,
+    gain=.02,
+    stimulus_description="N/A",
+    resolution=-1.0,
+    conversion=1.0,
+    timestamps=None,
+    starting_time=None,
+    rate=100_000.,
```

Comment:
I did not know this was a thing in Python!

Reply:
pretty cool, huh?